OneSignalSDK/onesignal/src/main/java/com/onesignal/OSUtils.java

```diff
@@ -444,6 +447,18 @@ private static void openURLInBrowser(@NonNull Uri uri) {
       return Collections.newSetFromMap(new ConcurrentHashMap<T, Boolean>());
    }
 
+   // Creates a new Set<String> from a Set String by converting and iterating a JSONArray
+   static Set<String> newStringSetFromString(String setString) throws JSONException {
```

Comment:
`setString` should be something like `jsonArray` instead.

Reply:
good idea, on it